deps/SnoopCompile/snoop_bench.jl

```diff
@@ -0,0 +1,12 @@
+using SnoopCompile
+
+snoop_bench(
+    BotConfig(
+        "Plots",
+        os = ["linux", "windows", "macos"],
+        else_os = "linux",
+        version = ["1.3", "1.4", "nightly"],
+        else_version = "1.4",
```

Comment:
I think it is better to use the `yml_path` option to prevent conflicts.
```suggestion
		yml_path= "SnoopCompile.yml"
        else_os = "linux",
        else_version = "1.4",
```